src/Commands.js

```diff
@@ -36,7 +36,7 @@ Commands.help = function() {
     var tempOutDiv = $('#output');
     for(var i = 0; i < commands.length; i++) {
         var tempString = commands[i];
-        var tempDiv = $('<div></div>');
+        var tempDiv = $('<div/>');
```

Comment:
Not really. Just thought I'd be consistent with my newer code. I like this form better.